raiden/ui/web/src/app/components/channel-events/channel-events.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, ParamMap } from '@angular/router';
+import { Observable } from 'rxjs';
+import { map, tap } from 'rxjs/operators';
+import { ChannelInformation, EventsParam } from '../../models/event';
+
+
+@Component({
+    selector: 'app-channel-events',
+    templateUrl: './channel-events.component.html',
+    styleUrls: ['./channel-events.component.css']
+})
+export class ChannelEventsComponent implements OnInit {
+
+    public eventsParam: Observable<EventsParam>;
```

Comment:
Small nit-pick, but you could stick to the `$` suffix on observable variables

Reply:
I have to check if we can enforce it with some kind of lint rule.